marconi-chain-index/test-lib/Helpers.hs

```diff
@@ -247,3 +291,16 @@ bimSlotNo (C.BlockInMode (C.Block (C.BlockHeader slotNo _ _blockNo) _txs) _era)
 
 bimBlockNo :: C.BlockInMode mode -> C.BlockNo
 bimBlockNo (C.BlockInMode (C.Block (C.BlockHeader _slotNo _ blockNo) _txs) _era) = blockNo
+
+-- | Should probably move in `cardano-api`.
+txFeesExplicitInShelleyBasedEra
+    :: C.ShelleyBasedEra era
+    -> C.TxFeesExplicitInEra era
+txFeesExplicitInShelleyBasedEra shelleyBased =
```

Comment:
If you only accept Right from `C.txFeesExplicitInEra` (as done in e2e tests) then you don't need to maintain this case for future eras

Reply:
The reason I created this was exactly for not dealing with an Either in my code. Since I'm working with ShelleyBasedEra, then `txFeesExplicitInShelleyBasedEra` is total compared to `txFeesExplicitInEra`.